client/state/sites/reducer.js

```diff
@@ -34,20 +43,44 @@ export function items( state = {}, action ) {
 			return Object.assign( {}, state, {
 				[ site.ID ]: site
 			} );
-
+		case SITES_RECEIVE:
+			return keyBy( action.sites, 'ID' );
 		case DESERIALIZE:
 			if ( isValidStateWithSchema( state, sitesSchema ) ) {
 				return state;
 			}
-
```

Comment:
The battle saga for newlines continues bcc533194473c3f5e9c3b4bd5a661e3f90918b50 😆